templates/versafix-1/template-versafix-1.html

```diff
@@ -2717,8 +2717,8 @@
                                         <table border="0" cellpadding="0" cellspacing="0" align="center" class="block-border-wrapper-inner"
                                                style="background-color: #ffffff; -ko-background-color: @backgroundColor; -ko-attr-bgcolor: @backgroundColor;
                                                       border-width: 0px; -ko-border-width: @[blockStyle.borderWidth]px;
-                                                      border-style: none; -ko-border-style: @[blockStyle.borderType];
-                                                      border-color: #ff0000; -ko-border-color: @[blockStyle.borderColor];
+                                                      border-style: none; -ko-border-style: @[blockStyle.borderWidth gt 0 ? blockStyle.borderType : 'none'];
+                                                      border-color: #ff0000; -ko-border-color: @[blockStyle.borderWidth gt 0 ? blockStyle.borderColor : 'transparent'];
```

Comment:
this change is ok, but in previous commits about borders, you added an option 'none' to the border. Why is that?